src/sinol_make/util.py

```diff
@@ -108,6 +108,38 @@ def check(path):
 		return None
 
 
+def save_config(config):
+	"""
+	Function to save nicely formated config.yml
+	"""
+
+	order = [
+		"title",
+		"memory_limit",
+		"time_limit",
+		"override_limits",
+		"scores",
+		{
+			"key": "sinol_expected_scores",
+			"default_flow_style": None
+		}
+	]
+	config_file = open("config.yml", "w")
+
+	for field in order:
+		if isinstance(field, dict):
+			if field["key"] in config:
+				yaml.dump({field["key"]: config[field["key"]]}, config_file, default_flow_style=field["default_flow_style"])
+				del config[field["key"]]
```

Comment:
Is there a reason for removing the field?

Reply:
Yes, just before closing the file the rest of the config is saved. This could be helpful if there were some custom fields in the config.

Reply:
Then let's document this, this mechanism is not obvious.

Reply:
Added description 59b6fe14207b5514a4592dd7fab08ff228abe24d